src/client/Microsoft.Identity.Client/Platforms/Features/DesktopOS/Kerberos/LsaBufferSafeHandle.cs

```diff
@@ -1,31 +1,28 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using System;
-using System.Runtime.InteropServices;
-
-namespace Microsoft.Identity.Client.Kerberos.Win32
-{
-#if !(iOS || MAC || ANDROID)

```

Comment:
Not sure what the spacing problem is ... but there are no changes except removal of #if and namepsace rename